Feature/AdminNavBar

Inhockorea opened the pull request:
- Admin Navbar => in Material UI 
- updated Header of the container app. 
- clean up of this branch so that only code related to navbar would be in this branch	af2ac2b	Eric <ejho13@berkeley.edu>

i-mckinney commented on Container App/src/Components/Header/Header.js:
```diff
@@ -1,28 +1,135 @@
-    </nav>
-  </header>
-);
+const useStyles = makeStyles((theme) => ({
```
Lets find some time tomorrow to go over this. I want to see what we can pull out into a theme and have that as a separate file.

david-geisinger commented on Container App/src/Components/Header/Header.js:
```diff
@@ -1,28 +1,135 @@
+  },
+}));
+
+function Header() {
```
comment for high level component

Inhockorea commented:
Updated Header
1) Implemented app bar instead of bottomnav bar
2) Added documentations
3) cleaned up the import 
 	->import FeaturedPlayListIcon from "@material-ui/icons/FeaturedPlayList";
	   import AccountCircle from "@material-ui/icons/AccountCircle"; 
Above can not be summed in up in one object and must import icon one by one
